src/app/component/cbp-plan-feed/cbp-plan-feed.component.ts

```diff
@@ -60,13 +64,17 @@ export class CbpPlanFeedComponent implements OnInit {
   }
 
   showBottomSheet(): void {
+    // const bottomSheetRef = this.bottomSheet.open(FilterComponent,{
     this.bottomSheet.open(FilterComponent,{
       panelClass: 'filter-cbp',
```

Comment:
Is animation functionality working, this code should not impact the search functionalty